classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] As was pointed out in the previous review, the requirement below has still not been met:
  
   ---
   > - Make sure that when adding a student to a classroom it also sets the classroom for the student.
   ---
   
   Please consider the code suggestion below to help with this.
    ```suggestion
    def add_student(student)
        @students << student
        student.classroom = self
      end
    ```